doc/source/whatsnew/v0.24.0.txt

```diff
@@ -613,6 +613,7 @@ Categorical
 ^^^^^^^^^^^
 
 - Bug in :meth:`Categorical.from_codes` where ``NaN`` values in ``codes`` were silently converted to ``0`` (:issue:`21767`). In the future this will raise a ``ValueError``. Also changes the behavior of ``.from_codes([1.1, 2.0])``.
+- Constructing a :class:`pd.CategoricalIndex` with empty values and boolean categories was raising a ``ValueError`` after a change to dtype coercion in `78184868 <https://github.com/pandas-dev/pandas/commit/7818486859d1aba53ce359b93cfc772e688958e5>`_ (:issue:`22702`).
```

Comment:
you don't need to be so explict here (e.g. the commit is not really necessary)

Reply:
Will remove it. This is a remnant from when I had this under the `0.23.5.txt` release notes under "regressions fixed".